hasher-matcher-actioner/hmalib/lambdas/pdq/pdq_hasher.py

```diff
@@ -30,26 +30,6 @@
 ]  # Misnamed, this is a prefix, not a key, if renaming, use IMAGE_PREFIX
 
 
-def get_image_bytes(
```

Comment:
Moved this code into the `S3BucketImageSource`, `URLImageSource`, and `S3LocalBucketImageSource` objects to make use of patterns